proofs/cbmc/polyvec_matrix_expand/polyvec_matrix_expand_harness.c

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) The mldsa-native project authors
+// SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+
+#include "polyvec.h"
+
+void harness(void)
+{
+  polyvecl *mat;
+  const uint8_t *rho;
```

Comment:
This const isn't required. Let's remove it to be consistent with the other proofs.